package.json

```diff
@@ -100,7 +100,7 @@
     "multi-stage-sourcemap": "0.2.1",
     "platform": "1.3.0",
     "promise": "7.0.3",
-    "protractor": "2.2.0",
+    "protractor": "2.5.1",
```

Comment:
`protractor` bumped to resolve `selenium-webdriver` dependency (https://github.com/angular/protractor/blob/master/CHANGELOG.md#240) to let it be built without issues on node v4.